packages/wrangler/src/pages/build.tsx

```diff
@@ -164,7 +165,14 @@ export async function buildFunctions({
 		baseURL,
 	});
 
-	if (config.routes && routesOutputPath) {
+	if (!config.routes || config.routes.length === 0) {
+		throw new FatalError(
```

Comment:
We'll need to change the build image so that this doesn't kill a build entirely. It should just log this error and move on, imo.

Reply:
- [ ] Update PR to warn user and not output a worker instead of throwing an error

Reply:
Updated to warn for `pages publish` and `pages dev`, and exit with a special error code so we can warn + skip Functions in CI